### ISSUE TITLE
Need Help recovering Data

### ISSUE THREAD

**Issue by Viertelvor** [olivierfriard/BORIS#616]
I am using the latest Version for Windows (non-portable)

One of my projects froze for a while but seemed to still be working after which I could see everything but not do anything without an error showing (and also START and STOP of Behaviours did not show anymore). Since then, i can still load the project but once I do, I cannot do anything else anymore. I click 'Edit Project', 'Open Observation', 'Observation List' or anything and simply nothing happens. All my other projects still work so it's not the Program but the file.

Please Help 

[boris_error.log](https://github.com/olivierfriard/BORIS/files/11384460/boris_error.log)

**Comment by olivierfriard** [comment]
Hi,

the boris_error.log file reports that you are using an old version of BORIS (7.13.9). You should use the current version (8.17.1).
Send me your project file by email and I will check it

**Comment by Viertelvor** [comment]
Thank you so much! Yes the error occured with an older version but even with the new one nothing changed.

**Comment by olivierfriard** [comment]
Hi,
18446744075511.367 is a very high value for a time.
It does not correspond to an epoch time.
Can I ask you what is the time source f your coding?

**Comment by Viertelvor** [comment]
Yes, I think because the number is too high, that made the project crash.
I dont know what the time source of my coding is, maybe you have to dumb down the question.
If that info helps: I didnt do any coding in or with Boris.

**Comment by olivierfriard** [comment]
Hi,
from where does this value come if you did not code it?

I introduced a check to limit time stamp to -2147483647 and 2147483647 seconds

**Comment by Viertelvor** [comment]
I just recorded behaviours, then Boris froze and when it came back it gave me an error that looked like the one in #392 (stupidly i did not screenshot it so i cant be sure if it was the same) and when i closed and reopened Boris i could not access the project anymore. I found the number by opening the .boris file in notepad++ and looking through it manually. 

I'm sorry I really dont know much about coding, I hope that answers the question^^
